### PR TITLE
Landing page bottom section

### PR DESCRIPTION
**Pull request by dscreet**
**Figma**
image

**Implementation**
image

- Gaps between sections can be changed according to how other parts of the landing page are for consistency.
- Hover values are temporary until standardised.
- The right section of the footer is placed about mid height of left section. I followed the design but could place it to be on the same row as the privacy and tos text.
- Do you want me to compartmentalise both the contribute section and footer?

### CODE REVIEW

**Comment by Ellahinator** [comment]
To ensure styling consistency, I will suggest some changes.

With suggestions:
image

File: components/try.tsx [hunk 1]
```diff
@@ -13,28 +16,63 @@ export default function TryPear() {
         </div>
 
         {/* Content area */}
-        <div className="relative pb-10 pt-32 md:pb-16 md:pt-40">
+        <div className="relative pb-2 pt-32 md:pb-4 md:pt-40">
           {/* Section header */}
           <div className="mx-auto max-w-3xl pb-12 text-center md:pb-16">
-            <h1 className="h1 mb-4" data-aos="fade-up">
-              Try PearAI For Free
+            <h1 className="mb-4 text-4xl font-bold" data-aos="fade-up">
```
Comment (Ellahinator) [inline components/try.tsx]:
```suggestion
            <h1 className="mb-4 text-4xl font-semibold" data-aos="fade-up">
```
change the thickness of the header font to match figma and hero

File: components/try.tsx [hunk 2]
```diff
@@ -13,28 +16,63 @@ export default function TryPear() {
...
+            <p
+              className="text-secondary-400 mb-4 text-sm sm:text-lg"
+              data-aos="fade-up"
+              data-aos-delay="200"
+            >
+              Pear is built by a large community of developers. If you have
+              questions, or would like to discuss, you can join our{" "}
+              <Link
+                href="https://discord.gg/7QMraJUsQt"
+                className="hover:text-primary-800 text-primary-700"
```
Comment (Ellahinator) [inline components/try.tsx]:
```suggestion
                className="hover:text-primary-800 font-medium text-primary-700"
```
Makes the clickable text easier to read.

File: components/try.tsx [hunk 3]
```diff
@@ -13,28 +16,63 @@ export default function TryPear() {
...
+            </p>
+            <p
+              className="text-secondary-400 mb-8 text-sm sm:text-lg"
+              data-aos="fade-up"
+              data-aos-delay="200"
+            >
+              Help the community out by giving{" "}
+              <Link
+                href="https://github.com/trypear/pearai-app"
+                className="hover:text-primary-800 text-primary-700"
```
Comment (Ellahinator) [inline components/try.tsx]:
```suggestion
                className="hover:text-primary-800 font-medium text-primary-700"
```
Makes the clickable text easier to read.

File: components/try.tsx [hunk 4]
```diff
@@ -13,28 +16,63 @@ export default function TryPear() {
...
+            </p>
             <div className="mx-auto max-w-xs sm:flex sm:max-w-none sm:justify-center">
               <div data-aos="fade-up" data-aos-delay="400">
-                <a
-                  className="btn mb-4 w-full bg-primary-500 text-white-main hover:bg-primary-600 sm:mb-0 sm:w-auto"
-                  href="https://forms.gle/171UyimgQJhEJbhU7"
+                <Button
+                  asChild
+                  size={"lg"}
+                  className="hover:bg-primary-800 mb-4 w-full rounded-full bg-primary-700 text-white-main sm:mb-0 sm:w-auto"
```
Comment (Ellahinator) [inline components/try.tsx]:
```suggestion
                  className="hover:bg-primary-800 w-full rounded-full bg-primary-700 text-white-main hover:shadow-sm sm:w-auto"
```
button styling used from hero

File: components/try.tsx [hunk 5]
```diff
@@ -13,28 +16,63 @@ export default function TryPear() {
...
+                </Button>
               </div>
               <div data-aos="fade-up" data-aos-delay="400">
-                <a
-                  className="btn w-full bg-secondary-main text-white-main hover:bg-gray-800 sm:ml-4 sm:w-auto"
-                  href="https://docs.google.com/document/d/14jusGNbGRPT8X6GgEDbP1iab5q4X7_y-eFXK7Ky57IQ/edit?usp=sharing"
+                <Button
+                  asChild
+                  size={"lg"}
+                  className="w-full rounded-full border border-primary-700 text-primary-700 hover:bg-gray-100 sm:ml-4 sm:w-auto"
```
Comment (Ellahinator) [inline components/try.tsx]:
```suggestion
                  className="hover:border-primary-800 hover:text-primary-800 mt-2 w-full rounded-full border border-primary-700 text-primary-700 hover:shadow-sm sm:ml-4 sm:mt-0 sm:w-auto"

```
button styling used from hero

**Review by Fryingpannn — CHANGES_REQUESTED** [review]
Beautiful! Great job D. Last changes
- Align privacy policy, tos and the right side buttons. The all rights reserved part can be one line lower.
- For hovering, utilize what @Ellahinator did in his PR [here](https://github.com/sea-lene/pear-landing-page/pull/3/files#diff-ddc1067f080930bc48865f4660316a14b21bbc9f9ba85975ce8479342b983d2eR52). Basically, the primary button on hover will look slightly darker, whereas the secondary button on hover will have it's outline look darker (with slight shadow)

**Comment by dscreet** [comment]
> Beautiful! Great job D. Last changes
> 
> * Align privacy policy, tos and the right side buttons. The all rights reserved part can be one line lower.
> * For hovering, utilize what @Ellahinator did in his PR [here](https://github.com/sea-lene/pear-landing-page/pull/3/files#diff-ddc1067f080930bc48865f4660316a14b21bbc9f9ba85975ce8479342b983d2eR52). Basically, the primary button on hover will look slightly darker, whereas the secondary button on hover will have it's outline look darker (with slight shadow)

image

**Review by Fryingpannn — APPROVED** [review]

**Pull request merged**